inflearn/shopping-mall/server/src/resolvers/product.ts

```
@@ -14,4 +26,50 @@ export const productResolvers: Resolver = {
+    },
+    deleteProduct: (parent, { id }, { db }) => {
+      const existProductIndex = db.products.findIndex((item) => item.id === id)
+      if (existProductIndex < 0) {
+        throw new Error('없는 상품입니다.')
+      }
+
+      const updatedItem = {
+        ...db.products[existProductIndex],
+      }
+      delete updatedItem.createdAt
+      db.products.splice(existProductIndex, 1, updatedItem)
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)